README.md

```diff
@@ -686,59 +646,43 @@ ReactDOM.render(
 
 The options object for `createFarceRouter` should have a `historyProtocol` property that has a history protocol object. For example, to use the HTML History API as with `createBrowserRouter`, you would provide `new BrowserProtocol()`.
 
-The `createFarceRouter` options object does not have a default for the `render` property. It ignores the `renderPending`, `renderReady`, and `renderError` properties.
-
 The created `<FarceRouter>` manages setting up and providing a Redux store with the appropriate configuration internally. It also requires a `resolver` prop with the route element resolver object. For routes configured as above, this should be the `resolver` object in this library.
 
 #### `createConnectedRouter`
 
 `createConnectedRouter` creates a router that works with an existing Redux store and provider.
 
 ```js
-import {
-  Actions as FarceActions,
-  BrowserProtocol,
-  createHistoryEnhancer,
-  queryMiddleware,
-} from 'farce';
+import { BrowserProtocol, queryMiddleware } from 'farce';
 import {
   createConnectedRouter,
-  createMatchEnhancer,
-  createRender,
+  createFoundEnhancer,
   foundReducer,
-  Matcher,
   resolver,
 } from 'found';
 import { Provider } from 'react-redux';
-import { combineReducers, compose, createStore } from 'redux';
+import { combineReducers, createStore } from 'redux';
 
 /* ... */
 
 const store = createStore(
   combineReducers({
     found: foundReducer,
   }),
-  compose(
-    createHistoryEnhancer({
-      protocol: new BrowserProtocol(),
-      middlewares: [queryMiddleware],
-    }),
-    createMatchEnhancer(
-      new Matcher(routeConfig, { matchStemRoutes: false }),
-    ),
-  ),
+  createFoundEnhancer({
+    routeConfig,
+    matchOptions: { matchStemRoutes: false },
+    historyProtocol: new BrowserProtocol(),
+    historyMiddlewares: [queryMiddleware],
+  }),
 );
 
-store.dispatch(FarceActions.init());
+store.startRouting();
```

Comment:
i mean, i dunno... i think dispatching an action is pretty idiomatic. in this case that's all that's happening, no? i don't think redux add-ons in general follow this sort of pattern of binding action dispatches as methods onto the store